### ISSUE TITLE
Support Humidity sensors

### ISSUE THREAD

**Issue by rzr** [rzr/generic-sensors-lite#14]
This ticket is Following discussion with @dror3go:

https://github.com/rzr/generic-sensors-lite/issues/13

Basic support with htu21d has been tested from development branch:

https://github.com/rzr/generic-sensors-lite/commits/sandbox/rzr/devel/master

It was also tested along moziot webthing gateway:

https://github.com/rzr/mozilla-iot-generic-sensors-adapter/commits/sandbox/rzr/devel/master

[![htu21d](https://pbs.twimg.com/media/EOkS9pHW4AEnr9w?format=jpg#./file/htu21d.jpg)](https://twitter.com/RzrFreeFr/status/1218534773192364033# "htu21d")

Currently it's blocked by a couple of PR (cc: @bbx10)

https://github.com/bbx10/node-htu21d/pull/4

About Hygrometer API it's out of scope of W3C working group but 
@enchris from Intel OTC proposed something but it's gone:

https://github.com/intel/zephyr.js/issues/96

https://github.com/otcshare/hygrometer/blob/gh-pages/index.html

This could be useful if we want to redesign the project to handle several data types in the same object.

Anyway, non aligned sensors might be marked as experimental extras.

**Comment by rzr** [comment]
Hi @dror3go


As promised, some updates:

I am about to release generic-sensor-lite with htu21d support
please can you test master branch using node and iotjs ?

Then please open PR for htu21d as  temperature sensor

**Comment by rzr** [comment]
Would you be able to test a new version of addon ?

I plan to merge your patch:

https://github.com/dror3go/generic-sensors-lite/commit/6727023aa6420b645451780039b326f6d17b8521.patch

from:

https://github.com/rzr/generic-sensors-lite/issues/13

**Comment by rzr** [comment]
Please confirm it is working fine, I don't have those sensors, 

May we also support  ST's HTS221 Humidity and Temperature Sensor

https://www.st.com/resource/en/datasheet/hts221.pdf

Relate-to: https://github.com/rzr/mozilla-iot-generic-sensors-adapter/issues/13

**Comment by rzr** [comment]
I am considering to close this issue, may a new issue created about w3c alignments

Some hints at:

https://lists.w3.org/Archives/Public/public-device-apis/2020Jan/0004.html

https://github.com/intel/zephyr.js/issues/96
